README.md

```diff
@@ -323,10 +366,7 @@ page.uploadFile('#selector', '/path/to/file').then(function() {
 });
 ```
 
-## Advanced
-Methods below are for advanced users. Most people won't need these methods.
```

Comment:
I think if a developer is using this library is because know Node.js and have knowledge of PhantomJS, I can mark this warning in all functions that apply (pretty look)